mobile/library/cc/engine_builder.cc

```diff
@@ -737,6 +742,14 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
           ->add_canonical_suffixes(suffix);
     }
 
+    if (enable_port_migration_) {
+      alpn_options.mutable_auto_config()
+          ->mutable_http3_protocol_options()
+          ->mutable_quic_protocol_options()
+          ->mutable_num_timeouts_to_trigger_port_migration()
+          ->set_value(4);
```

Comment:
do we want to make this value configurable too?

Reply:
4 is the number that we found out to be the best in previous experiment.
I'd want EM users to simply enable port migration and not worry about the number of timeouts for now.
That said, if the need to experiment with port migration rises again, I'll be happy to add it.

I also intentionally added C++ API only because I want to verify the crash is fixed first.

Reply:
SGTM, maybe add a comment then about 4 being the best value from previous experimentation